### PR TITLE
Upgrade-instructions article.

### PR DESCRIPTION
**Pull request by vchimev**

### CODE REVIEW

**Comment by vchimev** [comment]
ping @ikoevska

File: upgrade-instructions.md [hunk 1]
```diff
@@ -33,26 +33,18 @@ tns platform update ios
...
+In order to use them in your old project, you will have to explicitly install the package, for example (assuming you are still in your main app project folder from the steps above):
 
-Windows:
 ```
-cd ..
-tns create TempApp
-rd /S /Q <yourapp>\app\tns_modules
-move TempApp\app\tns_modules <yourapp>\app
-rd /S /Q TempApp
+npm install --save tns-core-modules
```
Comment (Fatme) [inline upgrade-instructions.md]: It is better to be `npm install tns-core-modules --save`

File: upgrade-instructions.md [hunk 2]
```diff
@@ -33,26 +33,18 @@ tns platform update ios
 
 ### Upgrading TNS Modules
 
-Тhe next piece is a bit more tricky as it requires more steps. The easiest method is to create a new project and then copy the **tns_modules** out of the new project into your existing project and then delete the dummy project.
+Since 1.3.0 release **tns_modules** are available as a npm package named [tns-core-modules] (https://www.npmjs.com/package/tns-core-modules).
```
Comment (ikoevska) [inline upgrade-instructions.md]: Starting with NativeScript 1.3.0, the **tns_modules** are available as an npm package named [tns-core-modules](https://www.npmjs.com/package/tns-core-modules).

File: upgrade-instructions.md [hunk 3]
```diff
@@ -33,26 +33,18 @@ tns platform update ios
...
 
-Linux/Macintosh:
-```
-cd ..
-tns create TempApp
-rm -rf <yourapp>/app/tns_modules
-mv TempApp/app/tns_modules <yourapp>/app
-rm -rf TempApp
-```
+This command will install **tns-core-modules** package into node_modules folder and update package.json of the project by adding it as a dependency.
```
Comment (ikoevska) [inline upgrade-instructions.md]: This installs the **tns-core-modules** package to the node_modules folder and adds it as a dependency to the package.json of the project.

File: upgrade-instructions.md [hunk 4]
```diff
@@ -33,26 +33,18 @@ tns platform update ios
...
+
+> **NOTE:** tns_modules directory in your app folder will not be used anymore and you can safely remove it.
```
Comment (ikoevska) [inline upgrade-instructions.md]: The tns_modules directory

File: upgrade-instructions.md [hunk 5]
```diff
@@ -33,26 +33,18 @@ tns platform update ios
...
+
+Another place to find **tns-core-modules** package is [NativeScript Releases](https://github.com/NativeScript/NativeScript/releases/), where you can download tns-core-modules-\*.tgz containing the cross-platform modules from the respective release and execute the command above as giving it the local path of the package: `npm install --save <path to tns-core-modules-*.tgz>`.
```
Comment (ikoevska) [inline upgrade-instructions.md]: the tns... module

File: upgrade-instructions.md (hunk 1 above)
Comment (vchimev) [inline upgrade-instructions.md]: Done.

File: upgrade-instructions.md (hunk 2 above)
Comment (vchimev) [inline upgrade-instructions.md]: What do you think about _the cross-platform modules_ as tns_modules is just the name of the folder?

File: upgrade-instructions.md (hunk 3 above)
Comment (vchimev) [inline upgrade-instructions.md]: Done.

File: upgrade-instructions.md (hunk 4 above)
Comment (vchimev) [inline upgrade-instructions.md]: Done.

File: upgrade-instructions.md (hunk 5 above)
Comment (vchimev) [inline upgrade-instructions.md]: Done.

Comment (ikoevska) [inline upgrade-instructions.md]: Another place to find **tns-core-modules** package is [NativeScript Releases](https://github.com/NativeScript/NativeScript/releases/), where you can find a collection of the available tns-core-modules-*.tgz packages for every release. You can download a selected release and install it by running `npm install --save <path to tns-core-modules-*.tgz>`.

File: upgrade-instructions.md [hunk 6]
```diff
@@ -33,26 +33,18 @@ tns platform update ios
...
 
-Another way to achieve this is to go to [NativeScript Releases](https://github.com/NativeScript/NativeScript/releases/), download tns-core-\*.tgz containing the cross-platform modules from the respective release, unpack them and replace the content of the **tns_modules** folder in your project with them.
+> **IMPORTANT:** Since 1.3.0 release `tns create` command will create a new project, add **tns-core-modules** package as a dependency to its package.json and install it, so each new project you create will have the **tns-core-modules** package installed and you do not have to install it explicitly.
```
Comment (ikoevska) [inline upgrade-instructions.md]: Starting with NativeScript 1.3.0, the `tns create`... add the **tns-core-modules** package... So each new project you create will have the **tns-core-modules** package installed and you do not have to install it explicitly.

Comment (vchimev) [inline upgrade-instructions.md]: Done.

File: upgrade-instructions.md (hunk 2 above)
Comment (ikoevska) [inline upgrade-instructions.md]: +1 :)

File: upgrade-instructions.md (hunk 5 above)
Comment (vchimev) [inline upgrade-instructions.md]: Done.

File: upgrade-instructions.md (hunk 2 above)
Comment (vchimev) [inline upgrade-instructions.md]: Done.

**Pull request merged**